README.md

```diff
@@ -72,6 +72,12 @@ in the config file provided to Watchtower to avoid false positives due to
 Running watchtower behind a forward proxy is as simple as setting the
 `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables.
 
+### Deploying as a cloud.gov app
+
+To deploy watchtower as a cloud.gov app, run:
```

Comment:
Since the example command relies on variables in the example manifest, let's link them.
```suggestion
To deploy watchtower as a cloud.gov app using the [example manifest.yml file](manifest.yml), run:
```